vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -218,6 +254,7 @@ def fused_moe(
     topk: int,
     renormalize: bool,
     inplace: bool = False,
+    config: Dict[str, Any] = None,
```

Comment:
nit:
```suggestion
    config: Optional[Dict[str, Any]] = None,
```

BTW, why do we need this parameter?

Reply:
This is needed for tuning the kernel with the `benchmark_mixtral_moe.py` script. We can also call it `override_config` to make it a bit more explicit that this should not normally be used.